src/Arcus.Testing.Tests.Unit/Assert_/Fixture/TestCsv.cs

```diff
@@ -208,13 +232,27 @@ private static string GenValue()
 
         private static Func<string> GenValueFunc()
         {
+            CultureInfo cultureWithComma = new("nl-NL");
+            CultureInfo cultureWithDot = new("en-US");
+
             return Bogus.PickRandom(
                 () => Bogus.Lorem.Word(),
+                () => $"\"{RandomlyInsert(Bogus.Lorem.Sentence(), ",", ";", "%", "\\\"")}\"",
```

Comment:
Since we use randomized tests, adding this single line makes sure that we test for these special cases. We create a quoted sentence and insert randomly separator and escaped quote characters. And all together, it should still be considered a single text value.